addons/ui/org.openhab.ui.cometvisu/META-INF/MANIFEST.MF

```diff
@@ -52,5 +52,4 @@ Bundle-ClassPath: .,
  lib/sqlite-jdbc-3.8.7.jar
 Export-Package: org.openhab.ui.cometvisu.php
 Bundle-Activator: org.openhab.ui.cometvisu.internal.CvActivator
- 
- 
+Bundle-ActivationPolicy: lazy
```

Comment:
Hello @kaikreuzer mainly out of interest and to improve on reviewing :-)

I saw @sjka mentioning the lazy did not work well with our declarative usage of OSGI https://github.com/openhab/openhab2-addons/pull/2111/files#r109907790 are you aware of that and is that only in specific cases. Because some of the PR's which are open now also contain it. Is is a good practice a bad one or does it depend on specific situations?